Fixes/Updates

pshriwise opened the pull request:
This PR addresses a couple initial issues (#1 & #2) with the plotter after conversion to matplotlib and the `capi`. It also updates the README to a format more favorable for GitHub's markdown rendering. Finally, it applies a version to verify that saved plot views are compatible with the GUI rather than a default view.

paulromano commented on README.md:
```diff
@@ -1,133 +1,160 @@
 
-os, sys, pickle, copy, struct, threading, numpy, ast, PySide2
-openmc (plot-ids branch from Paul Romano)
+os, sys, pickle, copy, struct, threading, numpy, ast, PySide2, openmc
```
I wouldn't really call standard library modules "dependencies", since they are always present. The only real dependencies here are numpy, PySide2, and openmc.

paulromano replied:
Should add matplotlib too

pshriwise replied:
Good call. On it!

paulromano commented on plotmodel.py:
```diff
@@ -1,6 +1,7 @@
 import copy, struct, threading, openmc
 import openmc.capi.plot as capi_plot
 from openmc.capi.plot import _PlotBase
+import openmc
```
PEP8 note: you should separate imports into three sections:
1. standard library imports
2. third-party imports
3. local imports

each separated by a single line

paulromano commented on plotmodel.py:
```diff
@@ -11,6 +12,8 @@
 
+__VERSION__ = "0.1.0"
+
 _NOT_FOUND_ = -2
```
No need for trailing underscore

paulromano commented on plotgui.py:
```diff
@@ -106,12 +107,12 @@ def getIDinfo(self, event):
-        if yPos < self.model.currentView.v_res \
-            and xPos < self.model.currentView.h_res:
+        if yPos >= 0 and yPos < self.model.currentView.v_res \
+            and xPos >= 0 and xPos < self.model.currentView.h_res:
```
Could use chained operators here:
```Python
if 0 <= yPos < self.model.currentView.v_res \
    and 0 <= xPos < self.model.currentView.h_res:
```

pshriwise commented:
Thanks for the comments @paulromano! Hopefully I've addressed them all. 

I also added a shoutout to @landonjmitchell in the README for getting this going. Let me know if we should do that differently.
